pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
+
+`composer require {{user/package-name}}`
+
+- Install all dependencies for this project:
+
+`composer install`
+
+- Remove a package from this project:
```

Comment:
It does 2 things:

1. Uninstall the package
2. Removes the dependency from `composer.json` and `composer.lock`